syntax = "proto3";

package communication;

message Mission {
  uint32 drone_id = 1;
  string mission_data = 2;
}

message StatusUpdate {
  uint32 drone_id = 1;
  string status_message = 2;
}

message response{
    string text=1;
}
message Acknowledge {
  bool success = 1;
  string message = 2;
}
service CommunicationService {
  rpc SendMission (Mission) returns (response);
  rpc SendStatusUpdate (StatusUpdate) returns (response);
}
